CoreTableView/src/au/gov/asd/tac/constellation/views/tableview2/io/TableViewPreferencesIOUtilities.java

```diff
@@ -141,8 +146,11 @@ public static void savePreferences(GraphElementType tableType, final TableView<O
                 if (colSortNode.get(sortColumn).asText().equals("DESCENDING")) {
                     sortType = TableColumn.SortType.DESCENDING;
                 }
+                
+                // Extract page size details
+                pageSize = pageSizeNode.intValue();
```

Comment:
I managed a null pointer here by loading an old preference. Maybe check is necessary?

Reply:
Good pick up. Will add a null check for old preferences